src/ansys/aedt/core/modeler/cad/primitives.py

```diff
@@ -1481,6 +1481,58 @@ def cover_faces(self, assignment):
         self.oeditor.CoverSurfaces(["NAME:Selections", "Selections:=", obj_to_cover, "NewPartsModelFlag:=", "Model"])
         return True
 
+    @pyaedt_function_handler(selection="assignment")
```

Comment:
This is not needed because it is a new method, so you do not need to deprecate selection.

```suggestion
    @pyaedt_function_handler()
```